int16 left_x
int16 left_y
int16 middle_x
int16 middle_y
int16 right_x
int16 right_y
